handler/population-types.go

```diff
@@ -35,27 +43,67 @@ func (h *PopulationTypes) Get(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
-// Get is the handler for GET /area-types
+/*
+   GET population-type/<id>/area-types
+   Gets area types from cantabular.
+   If not authenticated, then only return
```

Comment:
This comment can probably be removed

Reply:
Again, given the roundabout way we are trying to get the unpublished datasets, i think this comment should definitely be there for future reference